Azeem/test/test_unmodLabel.py

```diff
@@ -0,0 +1,10 @@
+from unittest import TestCase
+from tex2Wiki import unmodLabel
+from tex2Wiki import setup_label_links
+class TestModLabel(TestCase):
+
+    def test_unmodLabel(self):
+        self.assertEqual('Formula:DLMF:25.2:E1', unmodLabel('Formula:DLMF:25.2:E1'))
+        self.assertEqual('Formula:DLMF:15.2:E1', unmodLabel('Formula:DLMF:15.02:E1'))
```

Comment:
From the test I get the impression that the funciton unmodLabel does nothing. Try to add better test cases, that it get's clear what the function does.